src/to.frontend/to.frontend/Factories/IApplicationState.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace to.frontend.Factories
+{
+    public interface IApplicationState
+    {
+        T Get<T>(string key);
+        void Set<T>(string key, T value);
+    }
+
+    public class ApplicationState : IApplicationState
```

Comment:
move application state implementation/interface to seperate files
check file location ( Factories ? )

Reply:
Moved to own file and Helpers directory